src/ComputationContainer/LogHeader/BUILD

```
@@ -7,5 +7,5 @@ cc_library(
     deps = [
         "@libspdlog//:spdlog"
     ],
-    visibility = ["//visibility:public"],
+    visibility = ["//Logging:__pkg__"],
```

```suggestion
    visibility = ["//visibility:private"],
```

and move to Logging/ package